submissions/pashapushak1996/html-css-popup/scss/_layout.scss

```diff
@@ -0,0 +1,237 @@
+@import "base";
+
+body {
+  background-color: $white;
+  color: $grey-2;
+}
+
+.wrapper {
+  min-height: 100%;
+}
+
+.container {
+  max-width: 1200px;
+
+  margin: 0 auto;
+}
+
+.header {
+  display: flex;
+  justify-content: flex-end;
+
+  padding: 20px;
+
+
+  &__list {
+    display: flex;
+    align-items: center;
+    gap: 20px;
+  }
+
+  &__link {
+    color: $grey-2;
+    text-decoration: none;
+
+    &:focus {
+      outline: none;
+      border-radius: 1px;
+      box-shadow: 0 0 0 2px $blue-1;
+    }
+  }
+
+  &__link_notify {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    width: 25px;
+    height: 25px;
+  }
+
+  &__profile {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    width: 25px;
+    height: 25px;
+
+    border-radius: 50%;
+  }
+}
+
+.header__list li:nth-child(3) {
+  position: relative;
+
+  input {
+    position: absolute;
+    top: 0;
+    left: 0;
+
+    height: 0;
+    width: 0;
+
+    -webkit-appearance: none;
+    appearance: none;
+  }
+
+  input:checked ~ .popup {
+    display: flex;
+  }
+
+  input:focus + label {
+    outline: none;
+    border-radius: 1px;
+    box-shadow: 0 0 0 2px $blue-1;
+  }
+
+  label {
+    cursor: pointer;
+  }
+
+  .app-icon {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    width: 25px;
+    height: 25px;
+  }
+
+}
+
+.popup {
+  display: none;
+  flex-direction: column;
+
+  width: 340px;
```

Comment:
Try to use relative units `rem` and `em`, it's a good practice nowadays. 
You can read articles [here](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Values_and_units), [here](https://www.freecodecamp.org/news/what-is-rem-in-css/) or any related to it's info

Reply:
I did this. Thanks for article!